### PR TITLE
Fix: Modal component

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Shift the earlier modal to drawer component and build the right modal component

Preview: https://deploy-preview-10--design-with-ira.netlify.app/pages/components/modal.html

### CODE REVIEW

File: pages/components/drawer.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    </a>
+    <ul>
+
+      <li class="nav-list-item"> <a href=/pages/about.html>About </a> </li> <li class="nav-list-item"> <a
+            href=/pages/docs.html>Documentation </a> </li> <li class="nav-list-item"> <a
+              href=/pages/installation.html>Installation </a> </li> <li class="nav-list-item"> <a
+                href="https://github.com/HarshnaNHaswani/Ira">GitHub</a></li>
+    </ul>
+  </nav>
+
```
Comment (surya-nemiwal) [inline pages/components/drawer.html:32]: Can use vscode extension Prettier to format code and remove extra empty line

File: scripts/demo.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+const classToggle = (component) => component.classList.toggle('hidden')
+
+if(btnModalSimple){
+  btnModalSimple.forEach(btn => {
+    console.log("clicked");
+    btn.addEventListener('click', () => classToggle(modalSimple))
+  });
+  console.log(btnModalSimple, modalSimple);
```
Comment (surya-nemiwal) [inline scripts/demo.js]: can remove console.log before merging

File: scripts/demo.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const modalStacked = document.querySelector('.modal-stacked')
+
+
+if (btnSnackbar) {
+  btnSnackbar.addEventListener('click', () => {
+      snackbars.forEach(snackbar => classToggle(snackbar));
+  })  
+}
+
+const classToggle = (component) => component.classList.toggle('hidden')
```
Comment (surya-nemiwal) [inline scripts/demo.js]: variable name is not so descriptive, can give better name for more readability

**Review by Rajitha04 — APPROVED** [review]
Try to use rem instead of ch, rest looks fine

**Review by surya-nemiwal — APPROVED** [review]
Except few small changes, everything looks Good from my side. Code nicely modularized.

**Comment by HarshnaNHaswani** [comment]
Thanks for your time and the review @surya-nemiwal  @Rajitha04. Will make the necessary changes

**Pull request merged**